lib/rdoc/ri/driver.rb

```diff
@@ -1523,12 +1531,16 @@ def setup_pager
 
     jruby = RUBY_ENGINE == 'jruby'
 
-    pagers = [ENV['RI_PAGER'], ENV['PAGER'], 'pager', 'less', 'more']
+    pagers = [ENV['RI_PAGER'], ENV['PAGER'], 'pager', 'less', 'more', 'more.com']
 
     pagers.compact.uniq.each do |pager|
       next unless pager
 
-      pager_cmd = pager.split(' ').first
+      if RbConfig::CONFIG['target_os'] =~ /mswin|mingw/
+        pager_cmd = pager.sub('Program Files', "\0").split(' ').first.sub("\0", 'Program Files').gsub('"', '')
+      else
+        pager_cmd = pager.split(' ').first
+      end
```

Comment:
It seems too fragile.

```suggestion
      pager_cmd = pager[/\A\s*(?:"\K[^"]+(?=")|\K\S+)/]}
```